packages/go/openapi/src/paths/cypher.saved-queries.id.permissions.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2024 Specter Ops, Inc.
+#
+# Licensed under the Apache License, Version 2.0
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+parameters:
+  - $ref: './../parameters/header.prefer.yaml'
+  - name: saved_query_id
+    description: ID of the saved query
+    in: path
+    required: true
+    schema:
+      type: integer
+      format: int32
+delete:
+  operationId: DeleteSavedQueryPermissions
+  summary: Revokes permission of a saved query from users
+  description: Revokes permission of a saved query from a given set of users
+  tags:
+    - Cypher
+    - Community
+    - Enterprise
+  requestBody:
+    description: The request body for revoking permission a saved query from users
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            user_ids:
+              type: array
+              items:
+                  $ref: './../schemas/model.components.uuid.yaml'
```

Comment:
I think the above should be 
```
type: array
items: 
  type: string
  format: uuid
```
instead. I say this because your `$ref: './../schemas/model.components.uuid.yaml'` has `readOnly: true` when your request body's field: `user_ids` isn't